Src/Support/GoogleApis.Auth.DotNet4.Tests/GoogleApis.Auth.PlatformServices.Tests_Net45.csproj

```diff
@@ -15,6 +15,7 @@
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
     <TargetFrameworkProfile />
+    <StartArguments>--wait</StartArguments>
```

Comment:
So this is just the default from Visual Studio, yes? In other words, if I'm running it from the command line - or if I choose to override this in VS - I can run without --wait, with --test=... etc. I'm pretty sure that's the case, but I thought it worth checking.